src/applications/personalization/profile-2/components/ProfileMobileSubNav.jsx

```diff
@@ -95,6 +95,8 @@ const ProfileMobileSubNav = ({
 
       if (justSwitchedToMobile) {
         setIsMobile(true);
+        const menuButtonHeight = getElementHeight(theMenu.current);
```

Comment:
If the page loads when the page is not mobile-width, then the height of the placeholder would never get properly set and then switching to mobile-width would result in the menu appearing to sit directly over the profile section title.